src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -13,4 +13,9 @@ Feature: Allergies Management
   @selenium
   Scenario: Remove No known allergies
     And a user clicks Remove No Known Allergy icon
-    Then the system displays unknown in the allergies table
\ No newline at end of file
+    Then the system displays unknown in the allergies table
+
+  @selenium
+  Scenario: Add New Allergy
```

Comment:
then this should be loading the page. add new allergy should rather check if the system does add the allergy